pyproject.toml

```
@@ -0,0 +1,76 @@
+[project]
+classifiers=[
+  "Operating System :: OS Independent",
+  "Intended Audience :: Science/Research",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  'Operating System :: Microsoft :: Windows',
+  'Operating System :: Unix',
+  'Operating System :: MacOS',
+  "Topic :: Scientific/Engineering",
+]
+
+dynamic = ["version"]
+keywords = ["unstructured grids"]
+description = "Xarray-styled package for reading and directly operating on unstructured grid datasets following UGRID conventions"
+license = {text = "Apache-2.0"}
```

Arguably, I think you should link to your license file here instead of just the text of the license name
```suggestion
license = {file = "LICENSE.txt"}
```